api/migrations/versions/04c602f5dc9b_update_appmodelconfig_and_add_table_.py

```diff
@@ -19,15 +24,28 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table('tracing_app_configs',
-    sa.Column('id', models.types.StringUUID(), server_default=sa.text('uuid_generate_v4()'), nullable=False),
-    sa.Column('app_id', models.types.StringUUID(), nullable=False),
-    sa.Column('tracing_provider', sa.String(length=255), nullable=True),
-    sa.Column('tracing_config', sa.JSON(), nullable=True),
-    sa.Column('created_at', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
-    sa.Column('updated_at', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
-    sa.PrimaryKeyConstraint('id', name='tracing_app_config_pkey')
-    )
+    conn = op.get_bind()
+    
+    if _is_pg(conn):
+        op.create_table('tracing_app_configs',
+        sa.Column('id', models.types.StringUUID(), server_default=sa.text('uuid_generate_v4()'), nullable=False),
+        sa.Column('app_id', models.types.StringUUID(), nullable=False),
+        sa.Column('tracing_provider', sa.String(length=255), nullable=True),
+        sa.Column('tracing_config', sa.JSON(), nullable=True), 
+        sa.Column('created_at', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+        sa.PrimaryKeyConstraint('id', name='tracing_app_config_pkey')
+        )
+    else:
+        op.create_table('tracing_app_configs',
+        sa.Column('id', models.types.StringUUID(), default=lambda: str(uuid4()), nullable=False),
```

Comment:
The default here will not take effect during the migration process. It is recommended to use the following configuration. 

```
server_default=sa.text("uuid()")
```